packages/tooltip/src/vaadin-tooltip-mixin.js

```diff
@@ -669,6 +669,7 @@ export const TooltipMixin = (superClass) =>
     /** @private */
     __updateContent() {
       this.__contentNode.textContent = typeof this.generator === 'function' ? this.generator(this.context) : this.text;
+      this.dispatchEvent(new CustomEvent('content-changed', { detail: { content: this.__contentNode.textContent } }));
```

Comment:
Checking `textContent` after setting it to always return a string if `null` or `undefined` was set.